content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -137,18 +137,19 @@ $ cat .dvc/cache/6f/db5336fce0dbfd669f83065f107551.dir
 
 That's how DVC knows that the other two cached files belong in the directory.
 
-## Run-cache
+## Run cache
 
 `dvc exp run` and `dvc repro` by default populate and reutilize a log of stages
 that have been run in the project. It is found in the `runs/` directory inside
-the cache (or [remote storage]).
+the cache (or [remote storage]) by default.
```

Comment:
Why `by default`? Is the path configurable?

Reply:
I added this phrase since it's also used in https://dvc.org/doc/user-guide/pipelines/run-cache . I guess you were referring to the cache dir in that case? I don't think the `/runs` part is configurable.

Reply:
> I guess you were referring to the cache dir in that case?

🤔 Probably. I think we should probably drop it in both places.